src/components/authorized/MyHomePage.js

```diff
@@ -47,7 +63,9 @@ export default class MyHomePage extends React.Component {
             <div>
                 {errorMessage}
                 <h1>Home</h1>
-                { this.state.loading ? <li>Loading</li> : <CustomerCommissionList userId={this.state.user.id}/>}
+                <button onClick={() => this.handleClick(true)}>Customer</button>
```

Comment:
handleClick is a bad method name, let's rename it to something more appropriate

Reply:
done